lib/tapioca/rbi_ext/model.rb

```diff
@@ -4,6 +4,7 @@
 module RBI
   class Tree
     extend T::Sig
+    include Tapioca::RBIHelper
```

Comment:
FWIW: this monkey patching was only created to avoid too much changes in the DSL compilers when we migrated from `parlour` to `rbi`, this should be refactored away.